rules-tests/Arguments/Rector/FuncCall/FunctionArgumentDefaultValueReplacerRector/Fixture/version_compare.php.inc

```diff
@@ -6,6 +6,7 @@ class VersionCompare
 {
     public function run()
     {
+        version_compare($a, $b, $c);
```

Comment:
Case where previously the rule would have been applied even though no code was changed

Reply:
Could you extract this one a to a new "skip_*" file?